.travis.yml

```diff
@@ -66,5 +66,8 @@ script:
       -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}
   - ninja
   - ./bin/dxc --help
-  - ./bin/dxc -T ps_6_0 ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv
+  - ./bin/dxc -T ps_6_0 -Fo passthru-ps.dxil ../tools/clang/test/CodeGenSPIRV/passthru-ps.hlsl2spv
```

Comment:
Please keep the line that is outputting the results to screen to make sure that doesn't regress. (in the past we had a case where output to console had a bug and resulted in a segfault).

Reply:
Ah. Gotcha. Sure!

Unrelated, the dxil comparison seems to fail. Perhaps they include something about the system built on or timestamp. They sure include a lot more than the spirv does. And I thought spirv was pedantic.